sm/pmp.c

```diff
@@ -252,7 +266,7 @@ int pmp_unset_global(int region_idx)
   /* We avoid any complex PMP-related IPI management
    * by ensuring only one hart can enter this region at a time */
 #ifdef __riscv_atomic
-  spinlock_lock(&pmp_ipi_global_lock);
+  pmp_ipi_acquire_lock();
   send_and_sync_pmp_ipi(region_idx, IPI_PMP_UNSET, PMP_NO_PERM);
   spinlock_unlock(&pmp_ipi_global_lock);
```

Comment:
Could you wrap this with pmp_ipi_release_lock() for symmetricity?

Reply:
Ah ya, I was thinking about doing that as well. I had forgot to add it in. I will do that rn.

Reply:
Done!